modules/Task/containers/TaskDetailsContainer.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useEffect, useState } from "react";
+import Link from "next/link";
+import { useRouter } from "next/router";
+import { Button, Flex, Text, Title } from "@mantine/core";
+import { IconArrowLeft } from "@tabler/icons-react";
+
+import { TTask } from "@/shared/typedefs/types";
+import { useTasksContext } from "@/shared/utils/TasksProvider/TasksProvider";
+import TaskDetails from "../components/TaskDetails";
+
+function TaskDetailsContainer() {
+  const router = useRouter();
+  const taskId = router.query.taskId?.toString();
+  const { tasks } = useTasksContext();
+  const [task, setTask] = useState<TTask>();
+  useEffect(() => {
+    if (taskId) {
+      setTask(tasks.find((item) => Number(item.id) === Number(taskId)));
+    }
+  }, [taskId, tasks]);
+  return (
```

Comment:
```suggestion
  const [task, setTask] = useState<TTask>();
  
  useEffect(() => {
    if (taskId) {
      setTask(tasks.find((item) => Number(item.id) === Number(taskId)));
    }
  }, [taskId, tasks]);
  
  return (
```

[NIT]: add line breaks for better readability.